include/lsst/meas/modelfit/CModel.h

```diff
@@ -238,8 +238,7 @@ struct CModelControl {
     LSST_CONTROL_FIELD(
         psfName,
```

Comment:
Hmm.  Maybe.  Field names like these are really sequences of namespace-like terms, so I'm not sure whether calling them a "prefix" is warranted (it feels like calling the directory part of a full filename a prefix).  Regardless, it's clear that the documentation string below needs to be improved.